test/mocks/http/mocks.h

```diff
@@ -554,6 +554,7 @@ class MockAsyncClientStream : public virtual AsyncClient::Stream {
   MOCK_METHOD(void, setWatermarkCallbacks, (DecoderFilterWatermarkCallbacks & callback),
               (override));
   MOCK_METHOD(void, removeWatermarkCallbacks, (), (override));
+  MOCK_METHOD(const StreamInfo::StreamInfo&, streamInfo, (), (const override));
```

Comment:
Can you add some basic test? Or maybe even easier, just access this in one of the ext_proc integration tests and verify it does what it's suposed to do?

Reply:
To add the test, I have to introduce ext_proc changes. I was hping to do that in separate PR.